src/qthotkey/meson.build

```diff
@@ -1,17 +1,18 @@
 if get_option('qt6')
-  qtx11extras_dep = dependency('qt6', version: qt_req, required: false, modules: ['X11Extras'])
+  qtprivate_dep = dependency('qt6', version: qt_req, private_headers: true, required: false, modules: ['Core', 'Gui'])
```

Comment:
We require Qt 6.0 globally but QX11Application exists only since 6.2. This version needs to be set explicitly here.

https://doc-snapshots.qt.io/qt6-dev/qnativeinterface-qx11application.html